packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1122,10 +1122,13 @@ class _TabBarViewState extends State<TabBarView> {
       _controller.animation.addListener(_handleTabControllerAnimationTick);
   }
 
+  List<Widget> ensureGlobalKeysForList(List<Widget> items) =>
```

Comment:
ensure is also a strange name since it never checks if a key is already present. `wrapWithKey` would probably be more accurate.